src/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ set(BUILD_RTCORE_SHARED_LIBRARY 1)
 
 if (BUILD_WITH_GIF)
 message("Building with GIF support")
-
+include_directories(AFTER ${EXTDIR}/giflib-5.1.9/)
```

Comment:
We should only add this include directory for GIF if PREFER_SYSTEM_LIBRARIES is OFF because the system version may be different than the externals version.